tabbycat/draw/manager.py

```diff
@@ -187,11 +186,17 @@ def get_relevant_options(self):
     def get_teams(self):
         """Get teams in ranked order."""
         teams = super().get_teams()
-        if self.round.tournament.pref('draw_pullup_restriction') == 'least_to_date':
-            annotate_npullups(teams, self.round.prev)
 
         metrics = self.round.tournament.pref('team_standings_precedence')
-        generator = TeamStandingsGenerator(metrics, ('rank', 'subrank'), tiebreak="random")
+        extra_metrics = []
+
+        if self.round.tournament.pref('draw_pullup_restriction') == 'least_to_date':
+            extra_metrics.append('npullups')
+
+        if self.round.tournament.pref('draw_pullup_restriction') == 'easy_draw':
+            extra_metrics.append('draw_strength_score')
+
+        generator = TeamStandingsGenerator(metrics, ('rank', 'subrank'), extra_metrics=extra_metrics, tiebreak="random")
```

Comment:
Actually, even when I try the fewest-pullups restriction, I get: "24 out of 24 teams don't have a 'npullups' attribute."

The standings generator doesn't add metrics as attributes, they'll still be in `standing.metrics['npullups']`. Did we want to add the attributes in lines 203-206 below?